pyproject.toml

```diff
@@ -17,19 +20,28 @@ classifiers=[
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 chardet = "^5.2.0"
-nltk = "^3.5"
+nltk = "^3.8.1"
 xmltodict = "^0.13.0"
 toml = "^0.10.0"
 
 [tool.poetry.dev-dependencies]
-pre-commit = "^3.3.3"
-pytest = "^7.4.0"
+pre-commit = "^3.7.0"
+pytest = "^8.1.1"
 
 [tool.poetry.scripts]
 philter_lite = 'philter_lite.main:main'
 
 [build-system]
-requires = ["poetry_core>=1.6.0"]
+requires = ["poetry_core>=1.9.0"]
```

Comment:
FYI: poetry-core 1.8.1 is the version that poetry 1.7.1 requires: https://github.com/python-poetry/poetry/blob/8fe16f82ed3d0379e9a187909203700c9fb332e7/pyproject.toml#L35